src/BSD-3-Clause.xml

```diff
@@ -26,8 +26,9 @@
             </item>
             <item>
                 <bullet>3.</bullet>
-                Neither the name of <alt match=".+" name="organizationClause3">the copyright holder</alt> 
-                 nor the names of its contributors may be used to endorse or promote products derived from this 
+				<alt match="(The\sname\sof\s.+may not)|(Neither\sthe\sname\sof\s.+nor\sthe\snames\sof\sits\scontributors\smay)" name="organizationClause3">
```

Comment:
I think we want literal spaces here instead of `\s`.  [Our docs reference POSIX EREs][1] and [POSIX EREs do not include `\s`][2].  POSIX EREs do include `[:space:]`, but we already rely on the matching engine to normalize whitespace, case, etc. before comparing with our templates.

Also, you seem to be using tabs to indent while the surrounding code uses spaces (although I'm still holding out hope for automatically formatting these, either via #432 or @zvr's [xmlindent][3]).

[1]: https://github.com/spdx/license-list-XML/blob/v3.1/schema/ListedLicense.xsd#L259-L267
[2]: http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap09.html#tag_09_04
[3]: https://github.com/zvr/xmlindent